plugin/config/config.php

```diff
@@ -25,4 +25,7 @@
         'user' => 'c1iservice',
         'pass' => 'c1iservice.Password'
     ],
+    'backup_restore.backup_key'    => 'iservice', // Backup file is displayed only if it contains this string.
```

Comment:
We must not restore backups of iservice_v2!
```suggestion
    'backup_restore.backup_key'    => 'iservice_v3', // Backup file is displayed only if it contains this string.
```